pyproject.toml

```diff
@@ -31,7 +31,7 @@ packages = [{ include = "upstash_vector" }]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-httpx = "^0.25.0"
+httpx = ">=0.24.1,<0.26.0"
```

Comment:
what about just `^0.24.1`, does it also work with langchain?

Reply:
Yep, they use `^0.24.1`, if that's ok we can use that too

Reply:
that should be ok us